libafl/src/stages/dump.rs

```diff
@@ -117,7 +117,7 @@ where
     #[inline]
     fn restart_progress_should_run(&mut self, _state: &mut Self::State) -> Result<bool, Error> {
         // Not executing the target, so restart safety is not needed
-        Ok(true)
+        Ok(false)
```

Comment:
This means the stage will never run, right? That's wrong then

Reply:
no see stages/mod.rs

Reply:
i use peform_all() instead of perform_restartable on normal run